src/main/java/com/algorand/algosdk/crypto/LogicsigSignature.java

```diff
@@ -39,6 +40,60 @@ public class LogicsigSignature {
     @JsonProperty("msig")
     public MultisigSignature msig;
 
+
+    private static boolean isAsciiPrintable(final byte symbol) {
+        char symbolChar = (char) (symbol & 0xFF);
```

Comment:
Is the masking necessary here? Would `(char) symbol` be sufficient?

Reply:
yea, `(char) symbol` is just sufficient

Reply:
actually, we don't even have to do the byte to char conversion, I am taking that out.